decidim/decidim
# Export debate comments

#### :tophat: What? Why?
For the Proposals component, admins can export the proposals' comments to a spreadsheet, but this feature is not available for the Debates component (which also has comments).

This PR allows admins to export the comments for a debates component.

#### :pushpin: Related Issues
- Related to [MetaDecidim Proposal \#15817](https://meta.decidim.org/processes/roadmap/f/122/proposals/15817)

#### Testing
- Visit the debates component for any participatory space
- Click on the blue "Export button"
- Click on one of the three options
- Check mail (localhost:3000/letter_opener)
- Download .zip
- Export
- Open file (csv, json or excel)

#### :clipboard: Checklist

### :camera: Screenshots

![Export debate comments](https://user-images.githubusercontent.com/8806781/100759459-e90c5800-33f0-11eb-9507-db08cda1085e.gif)

:hearts: Thank you!
